website/for_teachers.py

```diff
@@ -1874,3 +1878,10 @@ def _create_customizations(db, class_id):
     }
     db.update_class_customizations(customizations)
     return customizations
+
+
+def render_why_class(adventure):
+    return safe_format(
+        gettext('see_adventure_shared_class'),
+        class_name=adventure.get("why_class"),
+        creator=adventure.get('creator')),
```

Comment:
Currently, this produces the following message:
<img width="586" alt="Screenshot 2025-01-27 at 11 43 45" src="https://github.com/user-attachments/assets/7ea6afdb-4a53-4153-9e8f-91237d746717" />

I suspect the comma is not intentional and the message should appear without the parenthesis?